common/persistence/persistenceMetricClients.go

```diff
@@ -258,7 +259,28 @@ func (p *persistenceMetricsClientBase) updateErrorMetric(scope int, err error) {
 	}
 }
 
-func (p *persistenceMetricsClientBase) call(scope int, op func() error) error {
+func (p *persistenceMetricsClientBase) call(scope int, op func() error, domainName ...string) error {
```

Comment:
For extendability, instead of adding domainName, it's better to add metricsTag, so that if we want to add other tags in the future we don't need to change this